vertx-service-discovery-bridge-consul/src/test/java/io/vertx/servicediscovery/consul/ConsulServiceImporterTest.java

```diff
@@ -62,12 +62,12 @@ public void setUp() {
             JsonObject result = new JsonObject();
             services.forEach(object ->
                 result.put(object.getString("ServiceName"), object.getJsonArray("tags", new JsonArray())));
-            request.response().end(result.encodePrettily());
+            request.response().putHeader("X-Consul-Index", "42").end(result.encodePrettily());
```

Comment:
Is this header necessary?

Reply:
Yes. Consul send it on every response and `ConsulClient` expects it.

Reply:
Ok, thanks, I didn't know that. Gonna merge the PR.